docs/lib/analytics/fragments/android/getting-started/20_installLib.md

```diff
@@ -0,0 +1,7 @@
+Next add the following dependencies to your app `build.gradle`:
+
+```groovy
+implementation 'com.amplifyframework:core:1.0.0'
+implementation 'com.amplifyframework:aws-analytics-pinpoint:1.0.0'
+implementation 'com.amplifyframework:aws-auth-cognito:1.0.0'
+```
```

Comment:
Rewrite:

Expand **Gradle Scripts**, open **build.gradle (Module: app)**. You will already have configured Amplify by following the steps in the Project Setup walkthrough.

Add Predictions by adding these libraries into the dependencies block:

```groovy
dependencies {
    implementation 'com.amplifyframework:core:1.0.0'

    // Add these lines in `dependencies`
    implementation 'com.amplifyframework:aws-analytics-pinpoint:1.0.0'
    implementation 'com.amplifyframework:aws-auth-cognito:1.0.0'
}
```

Reply:
Rewrote